requirements/dev.txt

```diff
@@ -0,0 +1,2 @@
+black==19.3b0; python_version >= '3.6'
```

Comment:
better use the last version no? (no pinning)
It's not like it will break something

Reply:
assuming different version will not format different way, yes it will not break